react/javascript/demo-core-sdk/src/DemoCoreSDK.jsx

```diff
@@ -30,7 +30,11 @@ import { Dashboards, LookImage } from './demos'
 
 export const DemoCoreSDK = () => {
   return (
-    <ComponentsProvider>
+    <ComponentsProvider
+      themeCustomizations={{
+        colors: { key: '#1A73E8' },
```

Comment:
Blue buttons instead of purple

Reply:
I thought there was a predefined theme that we can use instead?

Reply:
It's in a private package and it also includes Google fonts which may or may not be desirable (think spartan).